packages/core/src/drivers/db/models/programa.js

```diff
@@ -121,9 +116,9 @@ const ProgramaSchema = {
     type: DataTypes.STRING,
     field: 'estudio_oferta_demanda',
   },
-  lineasGeneracionAplicacionConocimiento: {
+  lgac: {
```

Comment:
I think it will be better like it was before

Reply:
Ok !